src/ui/components/Rating/styles.scss

```diff
@@ -75,3 +76,13 @@
     background-image: url("~ui/components/Icon/img/star-half.svg");
   }
 }
+
+.Rating--small-by-user {
```

Comment:
Looks like the size is controlled elsewhere? So this could omit the size as I mentioned and just control the colour.